daemon/mgr/image.go

```diff
@@ -125,6 +128,16 @@ func (mgr *ImageManager) GetImage(ctx context.Context, idOrRef string) (*types.I
 	return mgr.cache.get(idOrRef)
 }
 
+// RemoveImage deletes an image by reference.
+func (mgr *ImageManager) RemoveImage(ctx context.Context, name string, option *ImageRemoveOption) error {
```

Comment:
We should remove the image from cache too .

Reply:
you are right, i will fix it